h/templates/help.html.jinja2

```diff
@@ -20,6 +30,10 @@
     <script async defer src="{{ embed_js_url }}"></script>
   {% endif %}
 {% endblock %}
+        <div class="numbered-list-item">
```

Comment:
Would be good to refactor these to use actual numeric lists (`ol`, `li`).

Reply:
Done. Had to make a couple of CSS adjustments, unsurprisingly.